### PR TITLE
Feat/call recording bot

### PR DESCRIPTION
**Pull request by tobi-bams**

### CODE REVIEW

**Review by kevkevinpal — APPROVED** [review]
I think this is a good start, I think we might need to talk a bit about how we're getting the specific clip. Are you just looping through the specified S3 bucket to see if there's a new clip, getting the clip then notifying stakwork?

**Comment by tobi-bams** [comment]
> I think this is a good start, I think we might need to talk a bit about how we're getting the specific clip. Are you just looping through the specified S3 bucket to see if there's a new clip, getting the clip then notifying stakwork?

Yes, we need to specify how we intend to get specific clips.
What was on my mind was maybe we would be using `call_id` as the name we use to store files on s3, then I keep querying s3 with that particular file name(`call_id`) till I get a response

so my request would look like this: `bucket_name.s3.amazonaws.com/folder_name/call_id.mp4`

**Review by Evanfeenstra — APPROVED** [review]
looks good!

File: src/builtin/callRecording.ts [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+                    .setAuthor('CallRecordingBot')
+                    .setDescription('Call was recorded successfully')
+                  message.channel.send({ embed })
+                  return
+                } else {
+                  throw `Could not store in stakwork ${sendFile.status}`
+                }
+              }
+              // If recording not found after specified time then it returns an error
+              if (timeActive === 180000 && !file.ok) {
```
Comment (kevkevinpal) [inline src/builtin/callRecording.ts:156]: is this just 3 minutes?

Comment (tobi-bams) [reply src/builtin/callRecording.ts]: This is for test, I just updated it with my recent push

**Comment by Evanfeenstra** [comment]
looks good to me!

**Review by kevkevinpal — APPROVED** [review]
reACK LGTM 🚀

**Pull request merged**